src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -752,6 +744,12 @@ def update_os_patch_configuration_sub_setting(self, patch_configuration_sub_sett
         try:
             # note: adding space between the patch_configuration_sub_setting and value since, we will have to do that if we have to add a patch_configuration_sub_setting that did not exist before
             self.composite_logger.log("Updating system configuration settings for auto OS updates. [Patch Configuration Sub Setting={0}] [Value={1}]".format(str(patch_configuration_sub_setting), value))
+
+            if value == '':
+                self.composite_logger.log_debug("We won't update the system configuration settings since new configuration value to update does not an acceptable value. [Patch Configuration Sub Setting={0}] [Value To Update={1}]"
```

Comment:
nit: Treat these as variables. Variable names don't have spaces.
Re: "[Patch Configuration Sub Setting={0}] [Value To Update={1}]"

Reply:
Done